libs/ui/elements/src/lib/metadata-quality/metadata-quality.component.html

```diff
@@ -1,27 +1,19 @@
-<div
-  *ngIf="metadataQualityDisplay"
-  class="mb-6 metadata-quality"
-  (mouseenter)="showMenu()"
-  (mouseleave)="hideMenu()"
->
-  <div class="min-w-[200px]" [class]="smaller ? 'leading-[8px]' : ''">
-    <gn-ui-progress-bar
-      (focus)="showMenu()"
-      (blur)="hideMenu()"
-      tabindex="0"
-      [value]="qualityScore"
-      type="primary"
-    ></gn-ui-progress-bar>
-  </div>
-  <div
-    class="absolute z-10 bg-white border border-black border-opacity-35 rounded-lg shadow-lg p-5 whitespace-nowrap"
-    [class]="isMenuShown ? 'block' : 'hidden'"
-  >
-    <div class="mb-4 font-bold" translate>record.metadata.quality.details</div>
-    <gn-ui-metadata-quality-item
-      *ngFor="let e of items"
-      [name]="e.name"
-      [value]="e.value"
-    ></gn-ui-metadata-quality-item>
-  </div>
+<div *ngIf="metadataQualityDisplay" class="mb-6 metadata-quality">
+  <gn-ui-popover [content]="popoverItems" theme="light-border">
+    <div class="min-w-[200px]" [class]="smaller ? 'leading-[8px]' : ''">
+      <gn-ui-progress-bar
+        tabindex="0"
+        [value]="qualityScore"
+        type="primary"
+      ></gn-ui-progress-bar>
+    </div>
+  </gn-ui-popover>
 </div>
+<ng-template #popoverItems>
+  <div class="mb-4 font-bold" translate>record.metadata.quality.details</div>
```

Comment:
I would add a bit of padding to the content by adding a div around it with a class with some padding. For example:`<div class="p-2 py-4">`

This will make the component look a bit nicer:
from:
![image](https://github.com/user-attachments/assets/f81cf9dc-0de6-4b5e-9fa7-a91ce51fcf9f)
to:
![image](https://github.com/user-attachments/assets/7740239e-9f2c-44b9-ad36-b891dc5b692f)

What do you think?

Reply:
padding added